### ISSUE TITLE
Trojan reported against win64 v3.10.1

### ISSUE THREAD

**Issue by mceachen** [sqlitebrowser/sqlitebrowser#1623]
Details for the issue
--------------------

#### What did you do?

Scanned latest windows build with virustotal: https://www.virustotal.com/#/file/2a04eceaf32d5a96a8a7d8a91f78fdd0bc8c44a5ae7f86cde568fee27d422d12/detection

#### What did you expect to see?

No viruses

#### What did you see instead?

"Zillya" is reporting a "Trojan.Rozena.Win32.62258"

**Comment by chrisjlocke** [comment]
Imagine standing in a crowded bar and you try and hear what the barman is saying.  Is he saying "Windy", "Wednesday" or "Wendy" ?  Heuristic scanning is trying to find a pattern in a larger pattern, and depending on how good your hearing is (to return to the previous poor analogy) you sometimes get it wrong and report a false positive.  Thankfully you can turn to your mate and say, "what did he say?" for confirmation.

So in this instance, with 68 people listening in a crowded bar, 1 misheard what was going on.  If the other 67 are sure they heard "Wednesday" and not "Wendy", you can be pretty sure he wasn't saying "Wendy".

TL;DR- Sometimes virus engines mis-see patterns as those that belong to viruses (and not whole viruses) 
Its a false positive and can be ignored.

Normally we'd throw the executable to Zillya who can do a deeper analysis (and even look at the source code) to determine where the problem lies, which they can correct, but as we're heading to a 3.11.x release, we'll wait to see what happens with that executable.

**Comment by SilvioGrosso** [comment]
Hello @mceachen 

I have just downloaded myself, on Windows 10 (64 bit) the installer:
DB.Browser.for.SQLite-3.11.0-alpha1-win64
From here:
https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.11.0-alpha1

I have also downloaded the nightly build (DB.Browser.for.SQLite-win64)
From here:
https://nightlies.sqlitebrowser.org/latest/

In short, I have checked them with Kaspersky Total suite (18.0.0.405) 
No virus has been found on both the installers (alpha + nuightly).

See this screenshot with my results:

![kaspersky_total_security png](https://user-images.githubusercontent.com/33007383/48886373-07b3dd80-ee2c-11e8-808b-ef39982274d2.jpg)

**Comment by mceachen** [comment]
Thanks for the responses! I think that's reasonable to wait to the next minor release.

Thanks again for working on this project. It's really, really great.

**Comment by chrisjlocke** [comment]
[No engines out of 69 detected anything in the nightly builds](https://www.virustotal.com/#/url/04e4610bcad78514961e1c5214e76b1b609040f0264626f5bc63c30639452dbe/detection), so you can download that with confidence. [Nor with the alpha build](https://www.virustotal.com/#/url/07055eddb2f36f7af760d4293f0b738db90054ac28e0085c19a53f7a00bc33da/detection), which used one extra engine.  Yay - even better odds!

So yes, 'proof' that its a false positive with some wacky virus engine.

Thanks for reporting it though!  An extra thing to add to our checklist when we plop out our next release.

**Comment by justinclift** [comment]
Wonder if that means Zillya was getting things wrong too much?

Maybe they'll be added back to the list if it was just a temporary problem (eg with their scanner).

**Comment by chrisjlocke** [comment]
I've clicked 're-analyze' from the link @mceachen provided, and it does still show Zillya.  Submit a new file (as in the nightlies, in my above links) and Zillya doesn't show.  Weird.  Guess they have a 'pool' of engines and just pick/rotate as needed, depending on those which answer back in a timely fashion.  Or something.
